components/org.wso2.carbon.identity.keyrotation/src/main/java/org.wso2.carbon.identity.keyrotation/dao/DBConstants.java

```diff
@@ -24,8 +24,6 @@
 public class DBConstants {
 
     public static final int CHUNK_SIZE = 2;
-    public static final String YES = "YES";
```

Comment:
Let's make CHUNK_SIZE value configurable

Reply:
chunkSize is user-configurable with https://github.com/wso2/identity-tools/pull/22/commits/5789ff819be406845886b585a17a8dc5c148b2b0.